src/routes/PostDetail/Comments/Comments.js

```diff
@@ -29,6 +28,26 @@ export default class Comments extends Component {
     comments: []
   }
 
+  componentDidMount () {
+    this.ensureSelectedCommentPresent()
+  }
+
+  componentDidUpdate (prevProps) {
+    this.ensureSelectedCommentPresent()
```

Comment:
why do we want to do this every time? seems like we should at least put this behind a check if certain prop(s) changed

Reply:
the checks are contained in the called function